utils/zuora.js

```diff
@@ -144,6 +144,21 @@ class Zuora {
 		});
 	}
 
+	/**
+	 * Creating a stored credential profile within the created payment method
+	 * Zuora doc: https://knowledgecenter.zuora.com/Billing/Billing_and_Payments/LA_Hosted_Payment_Pages/B_Payment_Pages_2.0/H_Integrate_Payment_Pages_2.0#Add_a_Checkbox_for_Stored_Credential_Consent
+	 *
+	 * @param {String} agreementRef Reference for the customer consent agreement
+	 * @returns {boolean}
+	 */
+	setAgreement (agreementRef) {
+		const mitConsentAgreementSrc = 'External';
+		const mitProfileType = 'Recurring';
+		const agreementSupportedBrands = 'Visa,Mastercard';
+		const mitConsentAgreementRef = agreementRef;
+		return this.Z.setAgreement(mitConsentAgreementSrc, mitProfileType, agreementSupportedBrands, mitConsentAgreementRef);
```

Comment:
I think we usually have break before return (I wonder if we have a lint rule for this)
```suggestion

		return this.Z.setAgreement(mitConsentAgreementSrc, mitProfileType, agreementSupportedBrands, mitConsentAgreementRef);
```